index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+	<meta charset="UTF-8">
+	<title>Womarkers Code - Adail Sena</title>
+	
+</head>
+<body>
+	
+		<div id="menu">
+		<nav>
+		<p style="background-color:black;"</p>
+		
+			<img src="images/logo.png" alt="Logotipo do Womarkers Code" width:"200px" height="66px" >
+					
+            <a href="index.html" font color="#ffffff" text-decoration: none>HOME</font></a>
+			<a href="servicos.html" font color="#ffffff">SERVIÇOS</font></a>
+			<a href="portfolio.html" font color="#ffffff">PORTFOLIO</font></a>
+			<a href="sobremim.html" font color="#ffffff">SOBRE MIM</font></a>
+			<a href="contato.html" font color="#ffffff">CONTATO</font></a>
+			
+			</nav>
+		</div>
+	</div><!-- Fim do topo -->
+	<div id="banner">
+	<p style="background-color: #fb396E;"</p>
+	
+	<p>BEM VIND@</p>
+	<p>MEU PORTFÓLIO</p>
+	<p>Conheça o meu trabalho...</p>
+		<img src="images/Slider.png" alt="banner de fundo com quadrados">
+		<img src="images/slider01.png" alt="borboleta" width:"400px" height="340px">
+	</div>
+	
+	
+	<div id="servicos">
+	<p style="background-color:black;"></p>
+		<h2><font color="#fb396E"> Serviços</font></h2>
```

Comment:
Se já possui um elemento, como é o caso do `<h2>`, não há necessidade de colocar um outro elemento dentro para estilizar, bastando utilizar o próprio `<h2>` como base, exemplo:

```suggestion
		<h2 style="color:fb396E;">Serviços</h2>
```

Reply:
Lembrando que o ideal é fazer esse uso de CSS em uma folha de estilos separada e não em linha usando o atributo style. 🌶

Reply:
É uma boa prática respeitar a hierarquia dos Headings h1, h2, h3..., não usar só por causa do tamanho da fonte
```suggestion
		<h1 style="color:fb396E;"> Serviços</h1>
```